src/slicer.cpp

```diff
@@ -173,92 +172,101 @@ bool SlicerLayer::PossibleStitch::operator<(const PossibleStitch &other) const
     return false;
 }
 
-void SlicerLayer::connectOpenPolylinesImpl(Polygons& open_polylines, coord_t max_dist, coord_t cell_size, bool allow_reverse)
+std::priority_queue<SlicerLayer::PossibleStitch>
+SlicerLayer::findPossibleStitches(
+    Polygons& open_polylines, coord_t max_dist, coord_t cell_size, bool allow_reverse)
 {
-    // below code closes smallest gaps first
-
     std::priority_queue<PossibleStitch> stitch_queue;
 
+    int64_t max_dist2 = max_dist * max_dist;
```

Comment:
When I know I'm going to commit code which indents differently I first commit with wrong indentation and then make a separate commit for the indentation fix.

Git doesn't understand indentation, which makes for hard reviewing.

Just a little tip ;)